### ISSUE TITLE
New Feature:  Add dark mode functionality

### ISSUE THREAD

**Issue by chinmaym07** [EOS-uiux-Solutions/eos-icons-landing#47]
This will enhance the look & feel of the https://eos-icons.com/

**Comment by cyntss** [comment]
@chinmaym07 that would be amazing! I'll assign you to this issue (but if you don't want to work on it please let me know)

**Comment by sarthakkundra** [comment]
Hey @cyntss. I don't think there's active work happening on this issue. This is a big change that I think can only happen in chunks. Would love to discuss it with you.

I think we should start page-by-page and first start with the teams page since that has static data, it'll be the easiest to migrate. What do you think about it?

**Comment by vinayaksh42** [comment]
> Hey @cyntss. I don't think there's active work happening on this issue. This is a big change that I think can only happen in chunks. Would love to discuss it with you.
> 
> I think we should start page-by-page and first start with the teams page since that has static data, it'll be the easiest to migrate. What do you think about it?

Great idea @sarthakkundra, I agree that it will be much easier to solve this issue in smaller chunks. Do you want me to assign this issue to you?

**Comment by sarthakkundra** [comment]
Yeah sure but I'd like to have a thorough discussion to clarify the requirements before beginning work on this. Is there a periodic call that I can join? If not, can we set one up?

**Comment by vinayaksh42** [comment]
> Yeah sure @vinayaksh42 but I'd like to have a thorough discussion to clarify the requirements before beginning work on this. Is there a periodic call that I can join? If not, can we set one up?

Sure, let's schedule a call. Can you dm me on Slack so that we can fix a time for the call?

**Comment by A-K-2001** [comment]
Can i work on this issue,if it is available
